### DISCUSSION TITLE
brew install maven fails

### DISCUSSION THREAD

**Discussion by JosefSpringer** [Homebrew/discussions#865] [Everyday usage]
brew install maven 

--> results in
```
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
Error: No available formula or cask with the name "maven".
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.
josefspringer@AG-00713 ~ % brew gist-logs maven
Error: No available formula with the name "maven".
```

**Comment by jonchang** [comment]
Run `brew doctor` and try to fix all the problems that appear.

**Comment by JosefSpringer** [comment]
Hi Jonathan,

I have tried this:

brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some taps are not on the default git origin branch and may not receive
updates. If this is a surprise to you, check out the default branch with:
  git -C $(brew --repo homebrew/core) checkout master

josefspringer@AG-00713 ~ % brew install maven  
Updating Homebrew...
fatal: Could not resolve HEAD to a revision
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/cask).
==> Updated Casks
Updated 12 casks.

==> Searching for similarly named formulae...
Error: No similarly named formulae found.
Error: No available formula or cask with the name "maven".
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.

What can I try else?

Josef Springer
Senior Architect

esentri AG
"Wir vernetzen Systeme, Prozesse und Menschen."

Renatastraße 32
80634 München
Mobil: +49 171 5445918
Josef.Springer@esentri.com
www.esentri.com

Datenschutz
www.esentri.com/dsgvo
www.esentri.com/datenschutzerklaerung

Aufsichtsratsvorsitzender: Dr. Ullrich Buchacker
Vorstand: Frank Szilinski, Robert Szilinski (Vorstandsvorsitzender)
Sitz der Gesellschaft: Ettlingen, Amtsgericht Mannheim - HRB 715781

**Reply by carlocab** [reply]
Did you run the fix suggested by `brew doctor`?

**Comment by JosefSpringer** [comment]
Hi Carlo,

Thanks for your tip. I ran the suggested fix and what surprise I could install maven.
I had read the fix but I did not dare to run it, because I am new in this area and I did not understand exactly what I will change with this fix.
Many thanks!!

Josef Springer
Senior Architect

esentri AG
"Wir vernetzen Systeme, Prozesse und Menschen."

Renatastraße 32
80634 München
Mobil: +49 171 5445918
Josef.Springer@esentri.com
www.esentri.com

Datenschutz
www.esentri.com/dsgvo
www.esentri.com/datenschutzerklaerung

Aufsichtsratsvorsitzender: Dr. Ullrich Buchacker
Vorstand: Frank Szilinski, Robert Szilinski (Vorstandsvorsitzender)
Sitz der Gesellschaft: Ettlingen, Amtsgericht Mannheim - HRB 715781

**Comment by veniuscloudpower** [comment]
curl: (35) LibreSSL SSL_connect: SSL_ERROR_SYSCALL in connection to ghcr.io:443 

Error: maven: Failed to download resource "icu4c"
Download failed: https://ghcr.io/v2/homebrew/core/icu4c/blobs/sha256:5e416eb34196809126fe5465b776f355abbd4cee61c28e63d059a8f57e8503a6

or
curl: (35) LibreSSL SSL_connect: SSL_ERROR_SYSCALL in connection to ghcr.io:443 

Error: maven: Failed to download resource "harfbuzz"
Download failed: https://ghcr.io/v2/homebrew/core/harfbuzz/blobs/sha256:1bf9b5df00ba57b84584e03c86f44c491d34ca6c0c887c86d26afcad6df5155a
 

or 

curl: (35) LibreSSL SSL_connect: SSL_ERROR_SYSCALL in connection to ghcr.io:443 

Error: maven: Failed to download resource "xz"
Download failed: https://ghcr.io/v2/homebrew/core/xz/blobs/sha256:bdd5146737d8a84b8a855d7b1bdc81cd2d20692c5d77810fa7472cd9bb41fb47


For some time one or two out of three commands failed with exactly that error in any github repo.
